sfa_dash/static/css/styles.css

```diff
@@ -148,6 +148,7 @@ header nav{
 	width: inherit;
 }
 .data-metadata{
+    position:relative;
```

Comment:
indent and space after colon?

Reply:
Just went ahead and ran a linter on styles.css to fix lots of formatting issues.